Makefile

```
@@ -130,3 +130,10 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+code-generator:
+bash $(GOPATH)/src/k8s.io/code-generator@v0.21.5/generate-groups.sh \
```

Fix indentation, and does it really need bash here or just exection as `controller-gen` has it. Also how do I know I have this script locally?
```suggestion
    bash $(GOPATH)/src/k8s.io/code-generator@v0.21.5/generate-groups.sh \
```

---

2729a2449013ce01bb998ed635413d3b78585a2c fixed.
Yeah, the tool code-generator has to be exactly installed under go/src/k8s.io/ afaik.

---

Fix tab indent and add download logic for this.

---

Yes, it has been tab and add download task within Makefile. 
0ce90df6cde7f399781b319c16cfe1ce40cddfb6